service/src/main/resources/db/changesets/20220617_workspace_activity_log_table.yaml

```diff
@@ -0,0 +1,31 @@
+databaseChangeLog:
+- changeSet:
+    id: workspace activity log table
+    author: yuhuyoyo
+    changes:
+    - createTable:
+        tableName: workspace_activity_log
+        remarks: |
+          A table for logging change activities in the workspace
+        columns:
+        - column:
+            name: workspace_id
+            type: text
+            constraints:
+              nullable: false
+            remarks: |
+              The ID of the workspace
```

Comment:
Workspace UUID

(Nice to mention UUID, since workspaces also have user-facing ID)